src/app/shared/money-input/money-input.component.css

```diff
@@ -0,0 +1,32 @@
+input {
+  text-align: center;
+  background: #ffffff;
+  border: 1px solid #240b18;
+  box-sizing: border-box;
+  border-radius: 5px;
+  width: 164px;
```

Comment:
For consistency all numbers should be multiple of 4 or 5.
Do not mix it, choose only one: 4 or 5.
In case of 4 you should have border-radius: 4px; and width: 164px;.
In case of 5 you should have border-radius: 5px; and width: 165px;